OPHD/Things/Structures/SolarPlant.h

```diff
@@ -25,6 +26,9 @@ class SolarPlant : public PowerStructure
 
 	int calculateMaxEnergyProduction() override
 	{
-		return SOLAR_PLANT_BASE_PRODUCUCTION;
+		return static_cast<int>(SOLAR_PLANT_BASE_PRODUCUCTION / mMeanSolarDistance);
```

Comment:
I suppose if we were looking for mathematical accuracy, we should consider using the [inverse square law](https://en.wikipedia.org/wiki/Inverse-square_law).

----

Need to make sure you're not dividing by 0 here. I believe you set a default value of 0 for that field.

Reply:
I'll add the divide by 0 check.

I'm good with any answer concerning the inverse square law. I don't think it is worth too much thought beyond lets make it fun as there are probably dozens of other variables that could be considered? I was planning on leaving the actual balance for gameplay open and just add the feature. The solar distances are accurate in case we display those on planet selection the data would be correct.